readme.md

```diff
@@ -101,107 +102,18 @@ All these are bundled with this plugin, the ones ending with ```.umdl``` should
-              {"sensitivity": 0.5, "model_path": "hey_computer_alt.pmdl"}
-         ]
-    },
-    "hey_neon": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "hey_neon.pmdl"}
-         ]
-    },
-    "hey_device": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "hey_device.pmdl"}
-         ]
-    },
-    "hey_dexter": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "hey_dexter.pmdl"}
-         ]
-    },
-    "dexter": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "dexter.pmdl"}
-         ]
-    },
-    "ó_computador": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-             {"sensitivity": 0.5, "model_path": "ó_computador.pmdl"}
-         ]
-    },
-    "ó_jarbas": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-             {"sensitivity": 0.5, "model_path": "ó_jarbas.pmdl"}
-         ]
-    },
-    "ó_sauro": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-             {"sensitivity": 0.5, "model_path": "ó_sauro.pmdl"}
-         ]
-    },
-    "red_alert": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "red_alert.pmdl"}
-         ]
-    },
-    "thank_you": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "thank_you.pmdl"}
-        ]
-    },
-    "fuck_you": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "fuck_you.pmdl"}
-        ]
-    },
-    "hey_mycroft_snowboy": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "hey_mycroft.pmdl"}
-        ]
-    },
-    "wake_up_snowboy": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "wake_up.pmdl"}
-        ]
-    },
-    "desperta": {
-        "module": "ovos-ww-plugin-snowboy",
-        "models": [
-            {"sensitivity": 0.5, "model_path": "desperta_jm.pmdl"},
-            {"sensitivity": 0.5, "model_path": "desperta_jm2.pmdl"}
-        ]
     }
   }
 ```
 
-If you want to replace the default mycroft models without changing the wake words
+# Training your own wake word
 
-```json
- "listener": {
-      "wake_word": "hey_mycroft_snowboy",
-      "stand_up_word": "wake_up_snowboy"
- }
- 
-```
+You can spin up a docker container and train your model, this only works in x86
 
-# Training your own wake word
+`docker run -it -p 8000:8000 rhasspy/snowboy-seasalt`
+
+then navigate to http://localhost:8000
+
+Alternatively you can use the hosted instance at http://167.172.238.67:8000 and simply download your model  # TODO domain name
```

Comment:
Minor, but I would put the public API first and local second since it has additional requirements (docker on x86)